platform/source/mbed_retarget.cpp

```diff
@@ -689,6 +770,18 @@ extern "C" ssize_t write(int fildes, const void *buf, size_t length)
     }
 }
 
+#if MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY && DEVICE_SERIAL
+/* Write the contents of a buffer to a serial interface */
```

Comment:
"Write one character".

`DEVICE_SERIAL` define needs to be on the `#if` below, like for `getc`.